gcc/rust/checks/errors/borrowck/rust-bir-builder-pattern.h

```diff
@@ -90,7 +90,7 @@ class PatternBindingBuilder : protected AbstractBuilder,
   void visit (HIR::PathInExpression &expression) override {}
   void visit (HIR::QualifiedPathInExpression &expression) override {}
   void visit (HIR::RangePattern &pattern) override {}
-};
+}; // namespace Rust
```

Comment:
I think this needs to be removed? probably an automatic addition by clang-format due to mismatched brackets

Reply:
Rebases & fixed.